src/App.tsx

```diff
@@ -1,55 +1,68 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash/debounce';
```

Comment:
try to create your own debounce function

Reply:
ok, got it, i`ll fix this